### PR TITLE
Test request statistics

### PR DESCRIPTION
**Pull request by bratseth**
This is the first commit from https://github.com/vespa-engine/vespa/pull/14378 which just adds a test

### CODE REVIEW

File: jdisc_http_service/src/test/java/com/yahoo/jdisc/http/server/jetty/HttpServerTest.java [hunk 1]
```diff
@@ -959,23 +1003,38 @@ public ContentChannel handleRequest(final Request request, final ResponseHandler
...
+            ContentChannel responseContentChannel = ResponseDispatch.newInstance(Response.Status.OK).connect(handler);
+            responseContentChannel.write(ByteBuffer.wrap(parameters.toString().getBytes(StandardCharsets.UTF_8)),
+                                         NULL_COMPLETION_HANDLER);
 
             // Have the request content written back to the response.
             return responseContentChannel;
         }
     }
 
+    private static class RequestTypeHandler extends AbstractRequestHandler {
```
Comment (bjorncs) [inline jdisc_http_service/src/test/java/com/yahoo/jdisc/http/server/jetty/HttpServerTest.java:1021]: I cannot find use of this class outside unit tests

**Review by bjorncs — APPROVED** [review]

File: jdisc_http_service/src/test/java/com/yahoo/jdisc/http/server/jetty/HttpServerTest.java (hunk 1 above)
Comment (bratseth) [reply jdisc_http_service/src/test/java/com/yahoo/jdisc/http/server/jetty/HttpServerTest.java:1021]: It's in a test class so I hope not :-)

**Comment by bratseth** [comment]
@bjorncs could you take a look at the second commit as well?

File: jdisc_http_service/src/test/java/com/yahoo/jdisc/http/server/jetty/HttpServerTest.java (hunk 1 above)
Comment (bjorncs) [reply jdisc_http_service/src/test/java/com/yahoo/jdisc/http/server/jetty/HttpServerTest.java:1020]: Oops, my fault

**Review by bjorncs — APPROVED** [review]

**Pull request merged**